fix(curriculum): clarify instructions with implicit return (#54272)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6449842c6f6c84261075e4c9.md

Your `idToText` function should return the result of calling `.find()` on the `cells` array with a callback function that takes an `cell` parameter and returns `cell.id === id`.

> Note: your callback function should use an implicit return.

Reviewer: ```suggestion
Both of your functions should use implicit returns.
```

Does this seem more accurate?

I don't think it's necessary to have the `Note:` label. I'm not against it, it just seems like extra.
